### PR TITLE
MMT-2001 Adding cron to delete in work proposals last updated > 30 days ago.

### PR DESCRIPTION
**Pull request by ryanmiller-1**

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require File.expand_path(File.dirname(__FILE__) + "/environment")
```
Comment (ryanmiller-1) [inline config/schedule.rb]: Added to make the check for proposal mode

File: lib/tasks/proposals.rake [hunk 2]
```diff
@@ -0,0 +1,7 @@
+namespace :delete_proposals do
+  desc 'Delete proposals that are older than specified date old'
+  task :expired_proposals => :environment do
```
Comment (charleshuang80) [inline lib/tasks/proposals.rake]: `expired_proposals: :environment`

File: lib/tasks/proposals.rake [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    count = CollectionDraftProposal.delete_all("proposal_status = 'in_work' AND updated_at < '#{30.days.ago}'") if Rails.configuration.proposal_mode
```
Comment (charleshuang80) [inline lib/tasks/proposals.rake]: This selection was verified with the PO, right?

Comment (ryanmiller-1) [reply lib/tasks/proposals.rake]: Yes, another ticket was written to capture the work for other cases.

Comment (jceaser) [reply lib/tasks/proposals.rake]: There is no test for this, which may be hard to do, but I got a demo and I think the logic is good. updated_at < 30 days ago when looking at timestamp would work as expected.

**Pull request merged**